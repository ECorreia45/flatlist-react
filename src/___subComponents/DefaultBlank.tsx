import React from 'react';

const DefaultBlank = (): JSX.Element => (<p>List is empty...</p>);

export default DefaultBlank;
