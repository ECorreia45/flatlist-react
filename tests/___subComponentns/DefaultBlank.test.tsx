import React from 'react';
import DefaultBlank from './../../src/___subComponents/DefaultBlank';
import {render} from '@testing-library/react';

describe('DefaultBlank', () => {
    it('Should match snapshot', () => {
        const {asFragment} = render(<DefaultBlank/>);

        expect(asFragment()).toMatchSnapshot();
    });
});
