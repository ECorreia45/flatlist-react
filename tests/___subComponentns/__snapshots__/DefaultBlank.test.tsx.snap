// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefaultBlank Should match snapshot 1`] = `
<DocumentFragment>
  <p>
    List is empty...
  </p>
</DocumentFragment>
`;
